### ISSUE TITLE
Hogfather compatibily?

### ISSUE THREAD

**Issue by Floffyko** [turnermm/vkeyboard#1]
First of all, I cannot believe that I did not remarked this plugin in the last 10 Years... I discovered it finally and it is working nice In Hogfather DW version, (at least for me). Is any reason to mark it as incompatible with Hogfather?

**Comment by turnermm** [comment]
In order to use this with hogfather, you will have to turn off the feature flag in the configuration settings.  The only problem might be that I don't know how long that setting will be available.  I'll try to see if there's anything can do about it.

**Comment by Floffyko** [comment]
Yes, it seems I was lucky using Palemoon Browser. It seems in Chrome and Firefox with default settings is not working at al. Fact is that in DW Configuration settings 'vkeyboard' does not appear, but, quite for the moment being, I am happy having it working in Palemon. Windows, Linux, Mac language switch keyboard combination are always a trial and error business, while this plugin make the difference between working blind with different keyboard layouts and languages and something strait-ford and clear like this Plugin. The reality is that sometime the people, always in hurry, they don't realize which their needs are... but afterwards they ask themself how could they live without this plugin :-)
